test/integration/src/config-util.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
```

Comment:
is dotenv used?

We should remove it from the project if unused.

Reply:
I assume that these variables will be passed into the test from ci but in order to run them locally they need to be populated from somewhere. How would you suggest I deal with these locally?